### ISSUE TITLE
Add genome comparison using quast/mummer

### ISSUE THREAD

**Issue by danwiththeplan** [danwiththeplan/genomics_Lincoln#2]
Add a section on comparing genomes using quast/mummer using @elizabethpermina 's work.

**Comment by danwiththeplan** [comment]
@elizabethpermina currently I have this from you:

---

Comparing assemblies - mummer:
The mummer package has many capabilities for comparing complete genomes, contigs with genomes and contigs with contigs.  In the first instance, we can compare one of our assemblies with another, getting a graphical plot of the similarities.

```
nucmer --prefix=edena_velvet25 edena_contigs.fasta \
velvet_25/contigs.fa
```

which makes a file edena_velvet25.delta of the matching regions.

```
mummerplot edena_velvet25.delta -R edena_contigs.fasta \
-Q velvet_25/contigs.fa --filter --layout
```

plots this to a X11 window.
The completed Campylobacter jejuni is in the file CJ.dbs (a fasta file).  Your assemblies or some of the Precooked ones can be run using this file as a reference.  Put CJ.dbs first in the nucmer command and with the -R option for mummerplot.  It is worth comparing the mira assembly with CJ.dbs.

---

I was thinking to just include the mummer comparison in the interests of time. Is it OK to use this?

**Comment by elizabethpermina** [comment]
yeah, if you see it working - the point is to make a point about assembly version comparison

**Comment by elizabethpermina** [comment]
I have a SRR065736.sra  file which is 58M
Just pushed it into the root (should have put into the raw/, I suppose... ) Let me know if you think it's too big for this workshop.
